pages/docs/connect/index.html

```diff
@@ -0,0 +1,6 @@
+title: Connect to a Cluster
+author: Chris Ward
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ab hoc autem quaedam non melius quam veteres, quaedam omnino relicta. Vestri haec verecundius, illi fortasse constantius. Est, ut dicis, inquit; Restatis igitur vos; Universa enim illorum ratione cum tota vestra confligendum puto. Duo Reges: constructio interrete. Et quidem iure fortasse, sed tamen non gravissimum est testimonium multitudinis. Quae cum dixisset paulumque institisset, Quid est?
```

Comment:
What are we doing with this dummy text. This should be ready, when we publish it right?